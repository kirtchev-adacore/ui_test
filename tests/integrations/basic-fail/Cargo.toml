[package]
name = "basic_fail"
version = "0.1.0"
edition = "2021"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dev-dependencies]
ui_test = { path = "../../.." }
tempfile = "3.3.0"

[[test]]
name = "ui_tests"
harness = false

[[test]]
name = "json"
harness = false

[[test]]
name = "ui_tests_invalid_program"
harness = false

[[test]]
name = "ui_tests_invalid_program2"
harness = false

[[test]]
name = "ui_tests_diff_only"
harness = false

[[test]]
name = "ui_tests_bless"
harness = false

#@normalize-stdout-test: "(command: canonicalizing path `tests/actual_tests_bless/auxiliary/aasdflkjasdlfjasdlfkjasdf`\n\nfull stderr:\n).*" -> "${1}No such file or directory"
