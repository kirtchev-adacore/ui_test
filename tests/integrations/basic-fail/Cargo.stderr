Error: tests failed

Location:
    $DIR/src/lib.rs:LL:CC
error: test failed, to rerun pass `--test json`

Caused by:
  process didn't exit successfully: `$DIR/target/ui/1/tests/integrations/basic-fail/debug/deps/json-HASH` (exit status: 1)
Error: tests failed

Location:
    $DIR/src/lib.rs:LL:CC
error: test failed, to rerun pass `--test ui_tests`

Caused by:
  process didn't exit successfully: `$DIR/target/ui/1/tests/integrations/basic-fail/debug/deps/ui_tests-HASH` (exit status: 1)
Error: tests failed

Location:
    $DIR/src/lib.rs:LL:CC
error: test failed, to rerun pass `--test ui_tests_diff_only`

Caused by:
  process didn't exit successfully: `$DIR/target/ui/1/tests/integrations/basic-fail/debug/deps/ui_tests_diff_only-HASH` (exit status: 1)
Error: failed to parse rustc version info: invalid_foobarlaksdfalsdfj

Caused by:

Location:
    $DIR/src/config.rs:LL:CC
error: test failed, to rerun pass `--test ui_tests_invalid_program`

Caused by:
  process didn't exit successfully: `$DIR/target/ui/1/tests/integrations/basic-fail/debug/deps/ui_tests_invalid_program-HASH` (exit status: 1)
Error: tests failed

Location:
    $DIR/src/lib.rs:LL:CC
error: test failed, to rerun pass `--test ui_tests_invalid_program2`

Caused by:
  process didn't exit successfully: `$DIR/target/ui/1/tests/integrations/basic-fail/debug/deps/ui_tests_invalid_program2-HASH` (exit status: 1)
error: 5 targets failed:
    `--test json`
    `--test ui_tests`
    `--test ui_tests_diff_only`
    `--test ui_tests_invalid_program`
    `--test ui_tests_invalid_program2`
